packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -678,23 +635,17 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
                 client: Client,
                 message: Message<Payloads.ObjectSpawned>
             ) => {
-                // Check that this is the authoritative client.
-                if (client.authoritative) {
+                // Check that this is the authoritative client
+                const exclusiveUser = session.actorSet[message.payload.actors[0].id].exclusiveToUser;
```

Comment:
@stevenvergenz there's a regression here. `message.payload.actors` will be empty if the object failed to spawn (missing from library, glTF not found, etc.).